src/thok.rs

```diff
@@ -68,6 +71,33 @@ impl Thok {
         }
     }
 
+    pub fn save_results(&self) -> io::Result<()> {
+        let log_path = dirs::data_dir().unwrap().join("thokr.log");
+        // If the config file doesn't exist, we need to emit a header
+        let needs_header = !log_path.exists();
+
+        let mut log_file = OpenOptions::new()
+            .write(true)
+            .append(true)
+            .create(true)
+            .open(log_path)?;
+
+        if needs_header {
+            writeln!(log_file, "# Date, WPM, Accuracy, Standard Deviation")?;
+        }
+
+        writeln!(
+            log_file,
+            "{}, {}, {}, {}",
+            Utc::now(),
```

Comment:
```suggestion
            Local::now(),
```
since this is user facing I think local time would be more convenient